css/properties/text-decoration-skip.json

```
@@ -116,7 +106,40 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "≤13.1"
+                "version_added": "7"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "objects": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "7"
```

It was removed in https://github.com/WebKit/WebKit/commit/93c37f7d107452022ca972cd8107a0d6056ae93a ([613.1.3](https://github.com/WebKit/WebKit/blob/93c37f7d107452022ca972cd8107a0d6056ae93a/Source/WebKit/Configurations/Version.xcconfig#L24-L26) before [15.4](https://github.com/mdn/browser-compat-data/blob/54ff95bddf07953e311d304d1b133b5f2ecd28f9/browsers/safari.json#L202-L208)).

```suggestion
                "version_added": "7",
                "version_removed": "15.4"
```